Makefile

```diff
@@ -270,13 +278,23 @@ destroy-cumulus: cumulus-init
 force: ;
 
 # ---------------------------
+
+upload-cirrus-version:
+	git config --global --add safe.directory /CIRRUS-DAAC
+	git config --global --add safe.directory /CIRRUS-core
```

Comment:
I don't think that modifying the global git config from the Makefile is a good idea.